CMakeLists.txt

```diff
@@ -28,10 +28,8 @@ set(CMAKE_Swift_MODULE_DIRECTORY ${CMAKE_BINARY_DIR}/swift)
 
 # Build flags
 add_compile_definitions(
-    $<$<COMPILE_LANGUAGE:C,CXX>:U_ATTRIBUTE_DEPRECATED="">
     $<$<COMPILE_LANGUAGE:C,CXX>:U_SHOW_CPLUSPLUS_API=1>
     $<$<COMPILE_LANGUAGE:C,CXX>:U_SHOW_INTERNAL_API=1>
-    $<$<COMPILE_LANGUAGE:C,CXX>:U_STATIC_IMPLEMENTATION>
```

Comment:
This has broader implications on Windows. If this module is now being built dynamically, it needs to be explicitly packaged (see https://github.com/apple/swift-installer-scripts).

Is there a reason to build this dynamically now rather than statically? The only consumer of this library is Foundation right?

Reply:
Both CoreFoundation (which is a static library that is statically linked into the `Foundation` dynamic library) and `FoundationInternationalization` (a dynamic library) link this `_FoundationICU` module, so rather than statically linking ICU into both we took the approach. Would `FoundationEssentials` and `FoundationInternationalization` (which are new dynamic libraries that we are installing) also be in the same boat here?

Reply:
I took a look at that repo, looks like we'll need to update the installer scripts to account for the new `_FoundationICU.dll`, `FoundationEssentials.dll`, and `FoundationInternationalization.dll` libraries along with the new `FoundationEssentials.swiftmodule` and `FoundationInternationalization.swiftmodule`. That should be doable, I can post a PR to update it once we get far enough along in our toolchain builds to test it. Since we have to update it for FoundationEssentials and FoundationInternationalization anyways I don't think adding _FoundationICU to that list adds any additional burden - are there any other implications that we need to make sure we address here? (looks like we need to add the repos to the lists in the linux files too)